contribute/hacker-guide.md

```diff
@@ -160,20 +160,26 @@ Here are also some tips & tricks that have proven useful in Scala development:
 * Don't underestimate the power of `print`. When starting with Scala, I spent a lot of time in the debugger trying to figure out how
   things work. However later I found out that print-based debugging is often more effective than jumping around. While it might be obvious
   to some, I'd like to explicitly mention that it's also useful to print stack traces to understand the flow of execution. When working with `Trees`, you might want to use the Scala  `showRaw` to get the `AST` representation.
-* You can publish your newly-built scala version locally using `sbt publishLocal` to use it from `sbt`. Here's how:
+* You can publish your newly-built scala version locally using `sbt publishLocal`. Then, you can invoke a REPL using your version:
```

Comment:
you don't need to do `publishLocal` in order to run a REPL of your local version, just `sbt scala` (or `scala` within sbt) is enough.